keyboards/monsgeek/m3/info.json

```diff
@@ -0,0 +1,250 @@
+            { "flags": 4, "matrix": [2, 9],  "x":126, "y": 26},
+            { "flags": 4, "matrix": [2, 10], "x":140, "y": 26},
+            { "flags": 4, "matrix": [2, 11], "x":154, "y": 26},
+            { "flags": 4, "matrix": [2, 12], "x":168, "y": 26},
+            { "flags": 4, "matrix": [2, 13], "x":182, "y": 26},
+            { "flags": 4, "matrix": [2, 14], "x":196, "y": 26},
+            { "flags": 4, "matrix": [2, 15], "x":210, "y": 26},
+            { "flags": 4, "matrix": [2, 16], "x":224, "y": 26},
+
+            { "flags": 4, "matrix": [3, 0],  "x":  0, "y": 38},
+            { "flags": 4, "matrix": [3, 1],  "x": 14, "y": 38},
+            { "flags": 4, "matrix": [3, 2],  "x": 28, "y": 38},
+            { "flags": 4, "matrix": [3, 3],  "x": 42, "y": 38},
+            { "flags": 4, "matrix": [3, 4],  "x": 56, "y": 38},
+            { "flags": 4, "matrix": [3, 5],  "x": 70, "y": 38},
+            { "flags": 4, "matrix": [3, 6],  "x": 84, "y": 38},
+            { "flags": 4, "matrix": [3, 7],  "x": 98, "y": 38},
+            { "flags": 4, "matrix": [3, 8],  "x":112, "y": 38},
+            { "flags": 4, "matrix": [3, 9],  "x":126, "y": 38},
+            { "flags": 4, "matrix": [3, 10], "x":140, "y": 38},
+            { "flags": 4, "matrix": [3, 11], "x":154, "y": 38},
+            { "flags": 4, "matrix": [3, 13], "x":182, "y": 38},
+
+            { "flags": 4, "matrix": [4, 0],  "x":  0, "y": 51},
+            { "flags": 4, "matrix": [4, 1],  "x": 14, "y": 51},
+            { "flags": 4, "matrix": [4, 2],  "x": 28, "y": 51},
+            { "flags": 4, "matrix": [4, 3],  "x": 42, "y": 51},
+            { "flags": 4, "matrix": [4, 4],  "x": 56, "y": 51},
+            { "flags": 4, "matrix": [4, 5],  "x": 70, "y": 51},
+            { "flags": 4, "matrix": [4, 6],  "x": 84, "y": 51},
+            { "flags": 4, "matrix": [4, 7],  "x": 98, "y": 51},
+            { "flags": 4, "matrix": [4, 8],  "x":112, "y": 51},
+            { "flags": 4, "matrix": [4, 9],  "x":126, "y": 51},
+            { "flags": 4, "matrix": [4, 10], "x":140, "y": 51},
+            { "flags": 4, "matrix": [4, 13], "x":154, "y": 51},
+            { "flags": 4, "matrix": [4, 15], "x":182, "y": 51},
+
+            { "flags": 4, "matrix": [5, 0],  "x":  0, "y": 64},
+            { "flags": 4, "matrix": [5, 1],  "x": 14, "y": 64},
+            { "flags": 4, "matrix": [5, 2],  "x": 28, "y": 64},
+            { "flags": 4, "matrix": [5, 5],  "x": 70, "y": 64},
+            { "flags": 4, "matrix": [5, 9],  "x":126, "y": 64},
+            { "flags": 4, "matrix": [5, 10], "x":140, "y": 64},
+            { "flags": 4, "matrix": [5, 11], "x":154, "y": 64},
+            { "flags": 4, "matrix": [5, 13], "x":182, "y": 64},
+            { "flags": 4, "matrix": [5, 14], "x":196, "y": 64},
+            { "flags": 4, "matrix": [5, 15], "x":210, "y": 64},
+            { "flags": 4, "matrix": [5, 16], "x":224, "y": 64}
+        ]
+    },
+    "layouts": {
+        "LAYOUT": {
```

Comment:
```suggestion
    },
    "community_layouts": ["tkl_ansi"],
    "layouts": {
        "LAYOUT_tkl_ansi": {
```

Reply:
To confirm as part of a review request; would you agree the layout of this board matches that of the community layout "tkl_ansi"?

Reply:
:bump:

Reply:
@dunk2k  sure!